.github/ISSUE_TEMPLATE/feature_proposal.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Feature proposal
+about: Propose a great new feature for Mitiq
+title: ''
+labels: 'enhancement'
+assignees: ''
+
+---
+## Pre-Request Checklist
+
+- [ ] I checked to make sure that this feature has not already been requested
+- [ ] I or my organization would be willing to help implement and/or test this.
+- [ ] This is **not** a breaking change to Mitiq.
+
+## Feature description
+
+_TODO: Insert a short outline for what the proposed modification is._
+
+## Considerations
+
+_TODO: Explain why this modification to Mitiq is desirable. Briefly summarize the benefits and drawbacks of the chosen approach compared to other ways of achieving a similar functionality._
```

Comment:
```suggestion
_TODO: Explain the purpose of this feature and/or why this modification to Mitiq is desirable._
```

Reply:
Could you clarify the second sentence? Does "other ways of achieving similar functionality" mean using other tools that already exist? Or alternative ways I thought of doing this in Mitiq?

Reply:
Exactly as you suggest, its asking about are there other ways to fulfill the purpose of the proposed feature, either with other tools, approaches, or just even different ways in Mitiq to go about implementing it. Sometimes there is only one clear way, but it can be helpful when discussing a new feature to check and see if any other approaches meet the user need without major changes.